python/GafferSceneUI/TransformToolUI.py

```diff
@@ -101,6 +109,23 @@ def _distance( ancestor, descendant ) :
 
 	return result
 
+class _TargetTipWidget( GafferUI.Frame ) :
+
+	def __init__( self, tool, **kw ) :
+
+		GafferUI.Frame.__init__( self, borderWidth = 4, **kw )
+
+		labels = Gaffer.Metadata.value( tool, "ui:tool:transformToolTips" ) or []
+
+		with self :
+			with GafferUI.Frame( borderWidth = 0 ) as self.__innerFrame :
+				with GafferUI.ListContainer( orientation = GafferUI.ListContainer.Orientation.Vertical, spacing = 4 ) :
```

Comment:
A Frame, a ListContainer and N Labels seems a bit overkill here. Can we just use a single Label with `"\n".join( labels )`?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5558/commits/3aef5f2522e17d0c72a0026f2f6a9eebb45bd5c8